script-test/subtitles/imscsubtitles.js

```diff
@@ -275,16 +371,369 @@ require(
         });
       });
 
-      describe('example rendering', function () {
-        it('should call fromXML, generate and render when renderExample is called', function () {
-          subtitles = ImscSubtitles(mediaPlayer, { xml: '', text: stubResponse }, false, mockParentElement, {});
-          imscMock.fromXML.calls.reset();
+      describe('Live subtitles', function () {
```

Comment:
Does this make sense as a describe block given you're testing all of the functionality in here now?